packages/trader/src/Modules/Contract/Components/InfoBox/info-box-longcode.jsx

```diff
@@ -4,9 +4,10 @@ import classNames from 'classnames';
 import React from 'react';
 import { Icon, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 
 const InfoBoxLongcode = ({ contract_info }) => {
-    const max_longcode_length = 150;
+    const max_longcode_length = isMobile() ? 70 : 150;
```

Comment:
The width was reduced, please check if 70 is still relevant?

Reply:
it's 47 now, thank you for noticing